pom.xml

```diff
@@ -49,6 +49,7 @@
     <maven.build.timestamp.format>yyyyMMdd</maven.build.timestamp.format>
     <pentaho-reporting.version>8.2.0.0-SNAPSHOT</pentaho-reporting.version>
     <pdi-dataservice-client-plugin.version>8.2.0.0-SNAPSHOT</pdi-dataservice-client-plugin.version>
+    <pdi-dataservice-server-plugin.version>8.2.0.0-SNAPSHOT</pdi-dataservice-server-plugin.version>
```

Comment:
@nantunes Please use this name instead: `pdi-dataservice-plugin.version`
I know, I know the property names are reversed from the repository name :( :( :( 
https://github.com/pentaho/build-resources/blob/9.0.0.0/version.properties#L40